tools/rpm2img

```diff
@@ -320,20 +340,21 @@ fi
 cat <<EOF > "${BOOT_MOUNT}/grub/grub.cfg"
 set default="0"
 set timeout="0"
+set dm_verity_root="${DM_VERITY_ROOT[@]}"
 
 menuentry "${PRETTY_NAME} ${VERSION_ID}" {
    linux (\$root)/vmlinuz \\
        ${KERNEL_PARAMETERS} \\
        ${BOOTCONFIG} \\
        root=/dev/dm-0 rootwait ro \\
        raid=noautodetect \\
-       random.trust_cpu=on selinux=1 enforcing=1 \\
-       dm_verity.max_bios=-1 dm_verity.dev_wait=1 \\
```

Comment:
FWIW they *just* added [dm-mod.waitfor](https://www.kernel.org/doc/html/latest/admin-guide/device-mapper/dm-init.html) into the kernel so at some point in the future we may want to add this back in for the root partition once we are on 6.2+. This is especially useful for slower block devices that use probed asynchronously.